index.html

```diff
@@ -0,0 +1,25 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+</head>
+<body class="container">
+    <header class="toolbar">
+        <img src="./img/my-logo.png" alt="my logo" class="my-logo">
+        <button class="hamburger-menu"><img src="./img/hamburger.png" alt="Menu" id="hamburger-img"></button>
```

Comment:
- [ ] Please make sure that you use the `nav` tag to wrap your logo and the menu icon for semantic reasons. :relaxed: 
- [ ] Please :pray:  make sure that you use `a` tag to wrap your logo because this is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info. :blush: